### PR TITLE
Upgrade fezziwig

### PR DESCRIPTION
**Pull request by regiskuckaertz**
There is also a transitive dependency to libthrift in fezziwig, which I forgot to upgrade. After the upgrade, the compiler complained about two uses of `sequenceU` which is a method that was removed in cats 1.0. I replaced with `traverse` which does the transformation and accumulation in one loop.

### CODE REVIEW

File: atom-publisher-lib/src/main/scala/com/gu/atom/data/DynamoDataStore.scala [hunk 1]
```diff
@@ -145,14 +145,8 @@ abstract class DynamoDataStore
...
 
-  private def findAtoms(tableName: String): DataStoreResult[List[Atom]] = {
-    scan.flatMap { jsonItems =>
-      val atomDecoderResults = jsonItems.map { json =>
-        jsonToAtom(json)
-      }
-      atomDecoderResults.sequenceU
-    }
-  }
+  private def findAtoms(tableName: String): DataStoreResult[List[Atom]] =
```
Comment (regiskuckaertz) [inline atom-publisher-lib/src/main/scala/com/gu/atom/data/DynamoDataStore.scala]: Interesting that `tableName` isn't used at all, but I won't investigate that.

File: build.sbt [hunk 2]
```diff
@@ -7,7 +7,7 @@ name := "atom-maker-lib"
 
 lazy val baseSettings = Seq(
   organization := "com.gu",
-  scalaVersion := "2.11.11",
+  scalaVersion := "2.11.12",
```
Comment (mchv) [inline build.sbt]: `2.12` soon @regiskuckaertz ?

Comment (regiskuckaertz) [reply build.sbt]: 🤦‍♂️  of course, will add that!

Comment (regiskuckaertz) [reply build.sbt]: this is actually only used in atom-workshop which is still running on 2.11. I can look into it eventually ✌️

**Pull request merged**